_integration-test/run.py

```diff
@@ -0,0 +1,176 @@
+import subprocess
+import os
+from functools import lru_cache
+import requests
+import pytest
+import sentry_sdk
+import time
+import json
+
+SENTRY_CONFIG_PY = "sentry/sentry.conf.py"
+SENTRY_TEST_HOST = os.getenv("SENTRY_TEST_HOST", "http://localhost:9000")
+TEST_USER = "test@example.com"
+TEST_PASS = "test123TEST"
+TIMEOUT_SECONDS = 60
+
+
+def run_shell_command(cmd, **kwargs) -> subprocess.CompletedProcess:
+    return subprocess.run(cmd, shell=True, check=True, **kwargs)
```

Comment:
shell=True is an absolute no-go -- this is how you create security issues

this function shouldn't exist at all imo

Reply:
removed all instances of shell=True

Reply:
shell injection...ahh gotcha